kiwi/xml_parse.py

```diff
@@ -16,7 +16,7 @@
 #   kiwi/schema/kiwi_for_generateDS.xsd
 #
 # Command line:
-#   /home/ms/Project/kiwi/.tox/unit_py3_11/bin/generateDS.py -f --external-encoding="utf-8" --no-dates --no-warnings -o "kiwi/xml_parse.py" kiwi/schema/kiwi_for_generateDS.xsd
+#   /data/git/kiwi/env/bin/generateDS.py -f --external-encoding="utf-8" --no-dates --no-warnings -o "kiwi/xml_parse.py" kiwi/schema/kiwi_for_generateDS.xsd
 #
 # Current working directory (os.getcwd()):
 #   kiwi
```

Comment:
please drop this change from the pull request, as you only changed the schematron rule there are no real changes to the generated API except for the date and the python version you are using.

Reply:
overall I think we don't need a schema change, see my previous comments

Reply:
AFAICT we need a schema update no matter how it's implemented, as there needs to be some way to configure that.

Reply:
Would be nice if `generateDS.py` had an option to omit the header, or at least the system specific content. That way the CI could regenerate both kiwi.rng as well as `xml_parse.py` and check that they stay unchanged, to make sure they match the source files.